docs/specs/README.md

```diff
@@ -39,7 +39,7 @@ Create a new issue on our [GitHub repository](https://github.com/bigchaindb/bigc
 Submit a design specification document (Pull Request) of your feature. The spec needs to be submitted at:
 
 ```text
-/path/to/repo/bigchaindb/docs/specs/release-<latest-release>/
+/path/to/repo/bigchaindb/docs/specs/
```

Comment:
It seems that people have been _de facto_ using the `bigchaindb/proposals/` directory instead, so maybe you could just change it to that?